playbook.yml

```diff
@@ -117,6 +117,7 @@
       monit_protection: true
       nginx_install_method: package
       nginx_default_enable: false
+      nginx_client_max_body_size: "15M"
```

Comment:
I was reading https://github.com/ANXS/nginx and I think this variable should  be `client_max_body_size`. How did you verify this worked? 

And not a blocker, but why 15? Why not 10? Why not 100?